source/codegen/common_helpers.py

```diff
@@ -204,12 +204,14 @@ def get_input_and_output_custom_types(config, functions):
 
 def _get_nested_types(config, type_name):
     custom_types = set()
-    matching_custom_type = [
+    matching_custom_types = [
```

Comment:
Note: This change in `common_helpers.py` is more-so a fixup for the DC Power update. You can see the comment for these changes in that PR [here](https://github.com/ni/grpc-device/pull/744#discussion_r991607701).

That being said, I think it's good to have these changes in this PR as well to make sure the update isn't adversely affecting NI-SCOPE's generation with its new metadata.